src/types/env.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from 'zod';
+
+import { envSchema } from '@/../.env.validator';
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
```

Comment:
Maybe you can add a comment to explain why you disable the rule ?

Reply:
✅ Done.